src/domain/neighborhood/divisionSelector/__tests__/DivisionSelector.test.tsx

```diff
@@ -39,50 +47,20 @@ const defaultProps = {
   value: [],
 };
 
-const searchWord = 'saari';
-
-const filteredDivisions = [
-  'Karhusaari',
-  'Kulosaari',
-  'Lauttasaari',
-  'Mustikkamaa-korkeasaari',
-  'Talosaari',
-  'Vartiosaari',
-  'Vuosaari',
-];
-
-const divisions = fakeNeighborhoods(
-  divisionNames.length,
-  divisionNames.map((division) => ({
-    id: division,
-    name: {
-      fi: division,
-      en: division,
-      sv: division,
-    },
-  }))
-);
-
-const neighborhoodsResponse = {
-  data: { neighborhoodList: divisions },
-};
-
-const mocks = [
-  {
-    request: {
-      query: NeighborhoodListDocument,
-    },
-    result: neighborhoodsResponse,
-  },
-];
-
-const divisionId = divisions.data[0].id;
-const divisionName = divisions.data[0].name?.fi ?? '';
+beforeEach(() => {
+  server.use(
+    graphql.query('AdministrativeDivisions', (req, res, ctx) => {
+      return res(
+        ctx.data({
+          administrativeDivisions: divisions,
+        })
+      );
+    })
+  );
```

Comment:
You are declaring something very similar in `src/tests/msw/server-handlers.ts`. I'm assuming that I'm missing a difference in the logic.